### ISSUE TITLE
Willing to maintain django-jet ?

### ISSUE THREAD

**Issue by leogout** [geex-arts/django-jet#465]
@edmenendez GitHub has ability to add members of repository that can approve PRs and edit code. There is no need to add forks and confuse other developers.

_Originally posted by @f1nality in https://github.com/geex-arts/django-jet/issues/389#issuecomment-470218177_

Hello, in regard of this issue and this particular comment, I think it is appropriate for me to offer my time as a maintainer of django-jet. I love this admin interface and I have been working with it since 2018.
It saddens me to see so many pending PRs and unanswered issues, but it is justified as you have to work on your new SaaS platform.
As mentionned in the linked issue, you are willing to keep this repo open and accept community members work instead of getting forked.
I opened this issue for myself, but also to invite other developers to do the same if you answer positively. 

Thank you for your time and for this amazing platform. Have a nice day.

**Comment by edmenendez** [comment]
@leogout The issue is not if GitHub supports it. The issue is that it seems the repository is abandoned. GitHub's support for that is forking.

**Comment by leogout** [comment]
@edmenendez Yeah, yeah, I have read all your comments about how and why this repo is dead. No need to pollute this discussion with more of that.

Thank you for pointing me in the direction of a fork, although it is clearly not what I am asking. Also I can't see yours so why do you bother telling me what to do? 

This repo has 2.9k stars and a good reputation, it is nothing but common sense to ask to become a maintainer. I won't bother migrating the codebase to another fork, finding a new name for pip and restarting everything from the beginning again.

**Comment by edmenendez** [comment]
@leogout Sounds like you will be a fantastic maintainer if you can get someone to respond. I hope someone does. Good luck!

If you eventually choose to fork and need help finding a new pip name, I'll be happy to assist.

**Comment by f1nality** [comment]
Hi @leogout, nice to meet you! 

Thanks for your initiative and words, it can be really helpful since we still don't have enough resources to maintain this repository.
For sure we can add you as a maintainer if you are willing to support this repository.

We can discuss uploading to pip somewhere privately
My facebook https://www.facebook.com/d.kildishev/ or email denis@kildishev.ru

Are there anyone else who also willing to maintain this repository?

**Comment by edmenendez** [comment]
@f1nality I would be happy to help. I believe there are others listed in issue #389 that might also be interested.

**Comment by f1nality** [comment]
@edmenendez I've sent you invite to become maintainer as well, thanks a lot for supporting this project.

**Comment by leogout** [comment]
By the way, are you still selling Django Jet licences ? http://jet.geex-arts.com/prices 
If not, it may be the time to change the licencing system to attract more contributors / users. 
Also it is harder for me to be willing to spend my personnal time on this library, if you continue to make money with it.
I think @edmenendez would agree too.

What do you think @f1nality ?

**Comment by SalahAdDin** [comment]
@leogout They change the admin revenue mode and hence they discontinued this project.

**Comment by matwrob** [comment]
Hi @loegout, @edmendez, @salahaddin,
does anybody of you continue to work on it?

**Comment by SalahAdDin** [comment]
I left this long time ago.

**Comment by abhi1693** [comment]
If anyone's interested, here is a fork with maintenance fixes

https://github.com/Onemind-Services-LLC/django-jet

**Comment by assem-ch** [comment]
A version for supporting django 3 and 4: https://github.com/assem-ch/django-jet-reboot

**Comment by Barukimang** [comment]
@leogout great initiative!

Had made Django 3 and 4 versions of the package: https://github.com/Barukimang/django-jet

Here you can find them on Pypi:

https://pypi.org/project/django-3-jet/
https://pypi.org/project/django-4-jet/
